File: practice/src/main/java/com/decade/practice/DevelopmentApplication.java
package com.decade.practice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class DevelopmentApplication {

        public static void main(String[] args) {
                SpringApplication app = Application.createApp(DevelopmentApplication.class);
                app.setAdditionalProfiles("development");
                var context = app.run(args);
                Application.seeding(context);
        }
}

File: practice/src/main/java/com/decade/practice/DevelopmentApplication.kt
package com.decade.practice

import org.springframework.boot.autoconfigure.SpringBootApplication


@SpringBootApplication
class DevelopmentApplication

fun main(args: Array<String>) {
      val app = createApp(DevelopmentApplication::class.java)
      app.setAdditionalProfiles("development")
      val context = app.run(*args)
      initialize(context)
}